### DISCUSSION TITLE
Add external link to nav bar dropdown

### DISCUSSION THREAD

**Discussion by JB515** [alshedivat/al-folio#2361] [Q&A] [answered]
I've added a couple of external links to startups/open source projects my nav bar using the method described in #579 which is working well. I now need to add some more and as they're all to 'end user' stuff, I'd like to group them into a dropdown. I can make a dropdown for pages on my site, but this isn't working with external links. Is there a way to do this?

**Comment by george-gca** [comment, marked as answer]
I believe this can be easily done by creating a copy of [_pages/dropdown.md](https://github.com/alshedivat/al-folio/blob/master/_pages/dropdown.md) and inserting the links there. Just set the correct `permalink` attributes.
